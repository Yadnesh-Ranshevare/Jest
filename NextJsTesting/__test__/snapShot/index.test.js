import {render} from "@testing-library/react";
import Page from "../../src/app/Text/page";

test("snapshot test", () => {
    const container = render(<Page />);
    expect(container).toMatchSnapshot();
});