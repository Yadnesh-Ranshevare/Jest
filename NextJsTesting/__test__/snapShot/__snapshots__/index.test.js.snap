// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`snapshot test 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        This is a simple Next.js page for
        <img
          alt="simple image"
          src="https://www.google.com/url?sa=i&url=https%3A%2F%2Funsplash.com%2Fs%2Fphotos%2Fimage&psig=AOvVaw0Sgm7EqNKKkK-lMFYRA7jN&ust=1755776557604000&source=images&cd=vfe&opi=89978449&ved=0CBUQjRxqFwoTCOi1lsOnmY8DFQAAAAAdAAAAABAE"
          title="simple img1"
        />
        <input
          name="yadnesh"
          placeholder="enter your name"
          type="text"
        />
        <input
          type="text"
          value=""
        />
        <button>
          click me
        </button>
        <h1 />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      This is a simple Next.js page for
      <img
        alt="simple image"
        src="https://www.google.com/url?sa=i&url=https%3A%2F%2Funsplash.com%2Fs%2Fphotos%2Fimage&psig=AOvVaw0Sgm7EqNKKkK-lMFYRA7jN&ust=1755776557604000&source=images&cd=vfe&opi=89978449&ved=0CBUQjRxqFwoTCOi1lsOnmY8DFQAAAAAdAAAAABAE"
        title="simple img1"
      />
      <input
        name="yadnesh"
        placeholder="enter your name"
        type="text"
      />
      <input
        type="text"
        value=""
      />
      <button>
        click me
      </button>
      <h1 />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
